modules/registrars/openprovider/OpenProvider/API/API.php

```diff
@@ -223,6 +232,23 @@ protected function process(\OpenProvider\API\Request $r)
         return new \OpenProvider\API\Reply($ret);
     }
 
+    //Download from google drive and save to given path
+    public function downloadPHPfile($downloadURL, $savePath)
+    {
+        try {
+            $ch = curl_init();
```

Comment:
better to take this out of try catch and add a close statement in the catch as well

Reply:
Removed `downloadPHPfile()` function. Initially it was used this to download TLDs for Sandbox, but later the method was changed. No more useful that function!